contrib/lang/html/packages.el

```diff
@@ -96,3 +96,31 @@ which require an initialization must be listed explicitly in the list.")
 (defun html/init-slim-mode ()
   (use-package slim-mode
     :defer t))
+
+(evil-leader/set-key-for-mode 'web-mode
+  "mgp" 'web-mode-element-parent
+  "mgc" 'web-mode-element-child
+  "mgb" 'web-mode-element-beginning
+  "mgn" 'spacemacs/html-navigation-micro-state
+  "mgs" 'web-mode-element-sibling-next
+  ;; v like 'View'
+  "mvx" 'web-mode-dom-xpath
+  "mve" 'web-mode-dom-errors-show
+  ;; i like 'Item'
+  "mir" 'web-mode-element-rename
+  "miw" 'web-mode-element-wrap
+  "mic" 'web-mode-element-clone
+  "mf" 'web-mode-fold-or-unfold
+  ;; TODO element close would be nice but broken with evil.
+  )
+
+(spacemacs|define-micro-state html-navigation
```

Comment:
The navigation could be on `hjkl`:
- `h` previous element
- `l` next element
- `j` children
- `k` parent
- `H` previous sibling
- `L` next sibling
- `c` clone
- `r` rename
- `w` wrap
- `p` xpath